Structural/Registry/Tests/RegistryTest.php

```diff
@@ -6,6 +6,19 @@
 
 class RegistryTest extends \PHPUnit_Framework_TestCase
 {
+    public function testSimpleGetSet()
+    {
+        $key = 'myIdentifier';
+        $object = new \StdClass();
+        $object->props = array('a' => 1, 'b' => 2);
+        
+        Registry::set($key, $object);
+        $actual = Registry::get($key);
+        
+        $this->assertEquals($object, $actual);
```

Comment:
Why not extending the `testSetAndGetLogger` test case? It basically does the same thing except `assertEquals` on the get and set objects.